src/lib/mf.rs

```diff
@@ -370,7 +370,7 @@ impl<'a, TokId: PrimInt + Unsigned> MF<'a, TokId> {
         let la_tidx = self.parser.next_tidx(n.la_idx);
```

Comment:
Likewise.

Reply:
I agree with the other two; this one just makes the formatting almost impossible.